### PR TITLE
Flag the try-block to be abble not use the option nl_after_func_body

### PR DESCRIPTION
**Pull request by guy-maurel**
Ref. #1734

### CODE REVIEW

File: src/flag_parens.cpp [hunk 1]
```diff
@@ -38,6 +38,8 @@ chunk_t *flag_parens(chunk_t *po, UINT64 flags, c_token_t opentype, c_token_t pa
       if (  flags != 0
          || (parent_all && parenttype != CT_NONE))
       {
+         // Issue #1734
```
Comment (mwoehlke-kitware) [inline src/flag_parens.cpp]: Should this still be here?

File: src/uncrustify_types.h [hunk 2]
```diff
@@ -156,6 +156,7 @@ struct indent_ptr_t
 #define PCF_WF_ENDIF           PCF_BIT(39)  //! #endif for whole file ifdef
 #define PCF_IN_QT_MACRO        PCF_BIT(40)  //! in a QT-macro, i.e. SIGNAL, SLOT
 #define PCF_IN_FCN_CTOR        PCF_BIT(41)  //! inside function constructor
+#define PCF_IN_TRY_BLOCK       PCF_BIT(42)  //! inside Function-try-block
```
Comment (mwoehlke-kitware) [inline src/uncrustify_types.h]: Ack... aside from the conflicts with #2424, I also have a pending branch that needs a new flag :wink:.

File: src/flag_parens.cpp (hunk 1 above)
Comment (gmaurel) [reply src/flag_parens.cpp]: This is the next problem! The opening brace is not flaged. We have to look why.

**Comment by guy-maurel** [comment]
A new PR will be prepared.

File: src/newlines.cpp [hunk 3]
```diff
@@ -4882,9 +4886,13 @@ void do_blank_lines(void)
             }
             else if (options::nl_after_func_body() > 0)
             {
-               if (options::nl_after_func_body() != pc->nl_count)
+               // Issue #1734
+               if (!(pc->prev->flags & PCF_IN_TRY_BLOCK))
```
Comment (mwoehlke-kitware) [inline src/newlines.cpp]: BTW, now that #2424 is merged, this can be expressed more explicitly as `if (!pc->prev->flags.test(PCF_IN_TRY_BLOCK))`.

**Pull request closed without merging**